tests/test_parse_anything.py

```diff
@@ -15,7 +15,7 @@
 from spdx.parsers import parse_anything
 
 
-dirname = os.path.join(os.path.dirname(__file__), "data", "formats")
```

Comment:
Instead of replacing the old tests with the new ones, can we do both sets of tests?

Reply:
Hi Jeff, I added the formats test set back into test_parse_anything. Please tell me if there any other changes you would like me to make.